package.json

```
@@ -6,9 +6,11 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "aegir lint",
+    "build": "npm run build:proto && npm run build:proto-types && aegir build",
```

A prepare script to allow branches of this module to be used as gh url deps:

```suggestion
    "prepare": "npm run build",
    "build": "npm run build:proto && npm run build:proto-types && aegir build",
```